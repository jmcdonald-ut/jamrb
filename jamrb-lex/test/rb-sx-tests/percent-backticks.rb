%x(cd ~ && echo "HELLO WORLD!")
