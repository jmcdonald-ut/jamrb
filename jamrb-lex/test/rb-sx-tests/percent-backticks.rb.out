((Token ((1 0) on_backtick "%x("))
 (Token ((1 3) on_tstring_content "cd ~ && echo \"HELLO WORLD!\""))
 (Token ((1 30) on_tstring_end ")"))
 (Token ((1 31) on_nl "\n")))
